C++/Navio/Common/gpio.cpp

```diff
@@ -16,9 +16,8 @@
 #define HIGH                1
 
 /* Raspberry Pi GPIO memory */
-#define BCM2708_PERI_BASE   0x20000000
-#define BCM2709_PERI_BASE   0x3F000000
-#define BCM2835_PERI_BASE   0x3F000000
+#define BCM2708_PERI_BASE   0x3F000000
+#define BCM2711_PERI_BASE   0xFE000000
```

Comment:
This is not obvious to me why we removed one define and changed current values. We can move it to separate commit and add proper description.